clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/security/KubeconfigFileHasher.java

```diff
@@ -26,6 +26,9 @@
 public class KubeconfigFileHasher {
 
   public static String hashKubeconfigFile(String filepath) {
+    if (filepath == null || filepath.isEmpty()) {
```

Comment:
As there is only once caller and we control the input, can we decide whether an empty `filepath` should be `null` or `""` rather than check both (and annotate the parameter `@Nonnull` or `@Nullable` accordingly)?  Right now I think it can never be null as it always comes from `getKubeconfigFile`.

I've been trying to clean up some of the overly-defensive overly null-safe code in the Kubernetes provider 😄